backend/migration_service.py

```
@@ -270,4 +288,179 @@ async def cleanup_migration(self, migration_id: str) -> bool:
+                            if compose_file:
+                                stacks.append({
+                                    "name": item,
+                                    "compose_file": compose_file
+                                })
+                    except SecurityValidationError:
+                        # Skip invalid stack names silently
+                        continue
+            
+            return stacks
+        except Exception:
+            return []
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)